website/mails/mails.py

```diff
@@ -483,25 +483,25 @@ def get_english_article(word):
 
 REGISTRATION_BULK_UPLOAD_PRODUCT_OWNER = Mail(
     'registration_bulk_upload_product_owner',
-    subject='Here goes the email subject'
+    subject='Registry Could Not Bulk Upload Registrations'
 )
 
-REGISTRATION_BULK_UPLOAD_INITIATOR_SUCCESS = Mail(
-    'registration_bulk_upload_initiator_success',
-    subject='Here goes the email subject'
+REGISTRATION_BULK_UPLOAD_SUCCESS_ALL = Mail(
+    'registration_bulk_upload_initiator_success_all',
+    subject='Registrations Successfully Bulk Uploaded to your Community\'s Registry'
 )
 
-REGISTRATION_BULK_UPLOAD_INITIATOR_PARTIAL = Mail(
-    'registration_bulk_upload_initiator_partial',
-    subject='Here goes the email subject'
+REGISTRATION_BULK_UPLOAD_SUCCESS_PARTIAL = Mail(
+    'registration_bulk_upload_initiator_success_partial',
```

Comment:
Same here. I'd recommend to have the variable name of `Mail` instances the same as the template name just to avoid confusions.

Reply:
Done/fixed